### ISSUE TITLE
/proc mount verification fails when work path is a symlink

### ISSUE THREAD

**Issue by craftyguy** [postmarketOS/pmbootstrap#149]
When the work path is a symlink to some other directory (see example below), commands like `install` will fail the `ismount` test in mount.py since /proc/mounts references the mount by it's actual, non-symlinked pathname.

For example:

```
$ mkdir ~/work
$ ln -s ~/work ~/out
$ ./pmbootstrap.py init
...
[17:21:33] Work path []: ~/out
$ ./pmbootstrap.py install
[17:21:38] *** (1/5) PREPARE NATIVE CHROOT ***                                                                                                                                                                                                                                                                                                                                                                                                                                                   
[17:21:40] ERROR: Mount failed: /proc -> /home/craftyguy/out/chroot_native/proc 
$ mount |grep chroot
proc on /home/craftyguy/work/chroot_native/proc type proc (rw,nosuid,nodev,noexec,relatime) 

```

As you can see in the example, pmboostrap successfully mounts /proc, but the ismount verification fails since the symlink is resolved to the "real" path in the mount output.

**Comment by ollieparanoid** [comment]
Thank you for your report!

**Comment by ollieparanoid** [comment]
Fixed in 50d23e2fd56591446dfa5dca442b4c19f296e6ce.
